### DISCUSSION TITLE
Multi-Select Inputs

### DISCUSSION THREAD

**Discussion by JensRavens** [avo-hq/avo#782] [Feature requests]
This is kind of similar to https://github.com/avo-hq/avo/discussions/360, but slightly more general.

While tagging support on postgres arrays would be amazing, sometimes you have a select field that allows multiple values, but constrained to a fixed list of possible options (via the `<select multiple>` html tag, see https://www.w3schools.com/tags/att_select_multiple.asp).

**Comment by adrianthedev** [comment]
And how are you storing it in the database field? 

1. Have a string DB field and manually transform the field to & from an array?
2. Use a `t.string 'tags', array: true` field?
3. Use a `jsonb` postgres column?

**Reply by JensRavens** [reply]
the second option (that's why it's so similar to the linked feature request).

**Reply by adrianthedev** [reply]
I added it to the [roadmap](https://github.com/orgs/avo-hq/projects/3/views/1).

**Reply by adrianthedev** [reply]
[Tags field PR](https://github.com/avo-hq/avo/pull/847)

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/fields/select.html#multiple
